Lib/test/test_keyword.py

```diff
@@ -20,17 +20,34 @@ def test_changing_the_kwlist_does_not_affect_iskeyword(self):
         keyword.kwlist = ['its', 'all', 'eggs', 'beans', 'and', 'a', 'slice']
         self.assertFalse(keyword.iskeyword('eggs'))
 
+    def test_changing_the_softkwlist_does_not_affect_issoftkeyword(self):
+        oldlist = keyword.softkwlist
+        self.addCleanup(setattr, keyword, "softkwlist", oldlist)
+        keyword.softkwlist = ["foo", "bar", "spam", "egs", "case"]
+        self.assertFalse(keyword.issoftkeyword("spam"))
+
     def test_all_keywords_fail_to_be_used_as_names(self):
         for key in keyword.kwlist:
             with self.assertRaises(SyntaxError):
                 exec(f"{key} = 42")
 
+    def test_all_soft_keywords_can_be_used_as_names(self):
+        for key in keyword.softkwlist:
+            exec(f"{key} = 42")
+
     def test_async_and_await_are_keywords(self):
         self.assertIn("async", keyword.kwlist)
         self.assertIn("await", keyword.kwlist)
 
+    def test_match_and_case_are_soft_keywords(self):
+        self.assertIn("match", keyword.softkwlist)
+        self.assertIn("case", keyword.softkwlist)
+
```

Comment:
Add new line, maybe delete line above, since '_' is most likely (soft) keyword to accidentally omit. 
```suggestion
        self.assertIn("_", keyword.softkwlist)
```

Reply:
Oh, that's true, i forgot about `_` as a soft keyword. Thanks